Automattic/jetpack
# Repo Gardening: auto-label changes to the mu-wpcom package

## Proposed changes:

This should help automating labeling of PRs against the mu-wpcom package. We will not have to manually label changes to the "Launchpad" feature, for example.

### Other information:


## Jetpack product discussion

* No

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This is hard to test in this repo before we merge this. You can see it in a fork of this repo though:
https://github.com/jeherve/jetpack/pull/97
